yt_dlp/extractor/imgur.py

```diff
@@ -76,6 +76,24 @@ class ImgurIE(ImgurBaseIE):
             'thumbnail': 'https://i.imgur.com/jxBXAMCh.jpg',
             'dislike_count': int,
         },
+    },
+        # from https://github.com/yt-dlp/yt-dlp/issues/9458
+        {
+        'url': 'https://imgur.com/zV03bd5',
+        'md5': '4CB52A2B62E3C21EFF561DF0611C1E89',
+        'info_dict': {
+            'id': 'zV03bd5',
+            'ext': 'mp4',
+            'title': 'Ive - Liz',
+            'timestamp': 1710491255,
+            'upload_date': '20240315',
+            'like_count': int,
+            'dislike_count': int,
+            'duration': 56.0,
```

Comment:
use [`devscripts/run_tests.py`](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-support-for-a-new-site) and let it output the correct test data for you
```suggestion
            'duration': 56.92,
```

Reply:
my bad, didn't know it could do that - when i ran the tests i just saw xxx test cases passed, xxx skipped etc. and thought that was it

Reply:
It's all good, now you know. Thanks for the PR